src/modules/order/order.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, Entity, Index, JoinColumn, ManyToOne, OneToMany } from "typeorm";
+
+import { Entity as MedusaEntity } from "medusa-extender";
+import { Order as MedusaOrder } from "@medusajs/medusa";
+import { Store } from "../store/entities/store.entity";
+
+@MedusaEntity({override: MedusaOrder})
+@Entity()
+export class Order extends MedusaOrder {
+	@Index()
+	@Column({ nullable: true })
+	store_id: string;
+
+	@Index()
+	@Column({ nullable: false })
+	parent_id: string;
```

Comment:
`order_id`

Reply:
an order does not have necessarily an `order_id`, for example the parent one should not have itself.